core/src/main/java/hudson/Util.java

```diff
@@ -1479,6 +1479,23 @@ public static String intern(@CheckForNull String s) {
     }
 
     /**
+     * Convert a list of {@link StackTraceElement}s to a String with a new line for each element.
+     *
+     * @param elements List of elements
+     * @return String representation of the elements
+     */
+    @Nonnull
+    public static String stackTraceArrayToString(@Nullable StackTraceElement[] elements) {
```

Comment:
We can not use `Functions#printThrowable` instead, but those methods should be collocated and share common naming pattern.